packages/eslint-plugin-wpcalypso/lib/configs/recommended.js

```diff
@@ -10,6 +10,15 @@ module.exports = {
 	},
 	extends: [ 'eslint:recommended', 'plugin:jsdoc/recommended' ],
 	plugins: [ 'jsdoc', 'wpcalypso', 'inclusive-language' ],
+	settings: {
+		jsdoc: {
+			preferredTypes: {
+				// Allow both variations of object types.
```

Comment:
Any good reason for supporting both variations? Perhaps we should unify around a single one (I think `Object` is the consensus in the Gutenberg codebase).

Reply:
If that's the case, we can go with `Object`. The main reason was to avoid changing a bunch of code to fix it

Reply:
I split this off into another PR, because the change and autofix was quite large: https://github.com/Automattic/wp-calypso/pull/72348

Reply:
As a result, I removed the rule change from this PR.